assets/js/script.js

```diff
@@ -30,9 +31,11 @@ function initApi() {
       data.forEach(movie => {
         var card = document.createElement('div');
         card.setAttribute('class', 'card');
+        card.setAttribute("id", movie.id)
```

Comment:
Let's make this a data-id attribute instead of setting the id of the element itself.